modin/pandas/test/test_io.py

```diff
@@ -855,7 +888,10 @@ def test_read_csv_s3(self):
     @pytest.mark.parametrize("names", [list("XYZ"), None])
     @pytest.mark.parametrize("skiprows", [1, 2, 3, 4, None])
     def test_read_csv_skiprows_names(self, names, skiprows):
-
+        if Backend.get() == "Omnisci" and names is None and skiprows in [1, None]:
+            pytest.xfail(
+                "processing of duplicated columns in OmniSci backend is not supported yet"
```

Comment:
submit a tracking issue and reference it here

Reply:
Fixed.